docs/root/configuration/network_filters/mysql_proxy_filter.rst

```diff
@@ -15,6 +15,11 @@ as operations performed on each table.
    development. Capabilities will be expanded over time and the
    configuration structures are likely to change.
 
+.. warning::
+
+   The mysql_proxy filter was tested with MySQL v5.5. The filter may not work
```

Comment:
"may not work with"

Reply:
hmm, I'm quite sure we had verified this with mysql 8 at some point @venilnoronha how can I open  ticket ? will work on fixing this.

Reply:
@dkalani can you verify it with 8.x and update here? If it does work, we can update this message. If not, we can create an issue to track it and fix it.

Reply:
when we tried with upstream mysql (8 or 9), it didn't work properly. Thats why I asked venil to update this.